content/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -201,5 +201,8 @@ Since Terraform Cloud integrates with other systems, the permissions models of t
 
 - When a workspace is connected to a VCS repository, anyone who can merge changes to that repository's main branch can indirectly queue plans in that workspace, regardless of whether they have explicit permission to queue plans or are even a member of your Terraform Cloud organization. (And when auto-apply is enabled, merging changes will indirectly apply runs.)
 - If you use Terraform Cloud's API to create a Slack bot for provisioning infrastructure, anyone able to issue commands to that Slack bot can implicitly act with that bot's permissions, regardless of their own membership and permissions in the Terraform Cloud organization.
+- When a run task sends a request to an integrator, it provides an access token that provides access depending on the run task stage:
+  - For post-plan, it provides access to the runs plan json and the run task callback
+  - All access tokens created for run tasks have a lifetime of 10 minutes
```

Comment:
Just thinking;

Is it possible to be less specific about the `10 minutes` Just so we don't have to remember to change this document if we change the underlying code. e.g. `- All access tokens created for a task are short-lived and are only valid while the task is running`

Reply:
I did think about that, but the access token has a TTL that is defined separately to the run task timeout. Therefore technically, we could set a different time to the access token if that was something we needed.

Reply:
Fair enough.  That's raised another question in my head, but that's for the TFC repo ..